src/views/Register.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div :class="$style.container">
+    <div :class="$style.header">
+      <img :class="$style.logo" alt="wolox" src="@/assets/logo.png" />
+      <h1 :class="$style.title">{{title}}</h1>
```

Comment:
Always separate tags and properties of text to render
```suggestion
      h1.title
        |  {{ title }}
```